### PR TITLE
ui: update deps and minor rephrasing

### PR DESCRIPTION
**Pull request by ntarocco**
localhost_3000_profile

Help needed! I like the `Your activity` in the page, but I was not sure about `Your profile` in the menu: will you understand that you can find your loans in the profile? Isn't it normally about name/email changes?

### CODE REVIEW

**Comment by kpsherva** [comment]
snapshots are failing

File: ui/src/pages/frontsite/PatronProfile/PatronPastDocumentRequests/PatronPastDocumentRequests.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ export default class PatronPastDocumentRequests extends Component {
   renderEmpty = () => {
     return (
       <NoResultsMessage
-        messageHeader={'No past requests'}
-        messageContent={'You have no past literature requests.'}
+        messageHeader={'No past requests for new literatures'}
```
Comment (kpsherva) [inline ui/src/pages/frontsite/PatronProfile/PatronPastDocumentRequests/PatronPastDocumentRequests.js]: small: I was not sure if literature exists in plural, dictionaries say that it is uncountable https://dictionary.cambridge.org/dictionary/english/literature

File: ui/src/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js [hunk 2]
```diff
@@ -63,10 +63,9 @@ export default class PatronCurrentDocumentRequests extends Component {
   renderNoResults = () => {
     return (
       <NoResultsMessage
-        messageHeader={'Requests for new literature'}
+        messageHeader={'No requests for new literature'}
         messageContent={
-          'You did not create any request for literature, ' +
-          'that is not available at the library.'
+          "You don't currently have any request for new literature not available in the catalog."
```
Comment (topless) [inline ui/src/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js]: nit: I would avoid apostrophes. It needs a comma possibly after literature. What about

"You do not have any request for new literature."

File: ui/src/pages/frontsite/PatronProfile/PatronCurrentLoans/PatronCurrentLoans.js [hunk 3]
```diff
@@ -71,8 +71,8 @@ export default class PatronCurrentLoans extends Component {
     }
     return (
       <NoResultsMessage
-        messageHeader={'No loans'}
-        messageContent={'Currently you do not have any literature on loan'}
+        messageHeader={'No ongoing loans'}
+        messageContent={"You don't currently have any ongoing loan."}
```
Comment (topless) [inline ui/src/pages/frontsite/PatronProfile/PatronCurrentLoans/PatronCurrentLoans.js]:
```suggestion
        messageContent={"You do not have any ongoing loan."}
```

File: ui/src/pages/frontsite/PatronProfile/PatronMetadata/PatronMetadata.js [hunk 4]
```diff
@@ -1,18 +1,13 @@
...
 import { Header } from 'semantic-ui-react';
 
 export default function PatronMetadata(props) {
   return (
     <>
-      <Header as="h2">Welcome {props.user.username}!</Header>
+      <Header as="h2">Your activity</Header>
       <Header.Subheader>
-        You can check your library activity on this page
+        Here you can find your library activity.
```
Comment (topless) [inline ui/src/pages/frontsite/PatronProfile/PatronMetadata/PatronMetadata.js]:
```suggestion
        This is your library activity.
```

File: ui/src/pages/frontsite/PatronProfile/PatronPastLoans/PatronPastLoans.js [hunk 5]
```diff
@@ -116,7 +116,7 @@ export default class PatronPastLoans extends Component {
     return (
       <NoResultsMessage
         messageHeader={'No past loans'}
-        messageContent={'Currently you do not have any past loans'}
+        messageContent={"You don't have any past loan."}
```
Comment (topless) [inline ui/src/pages/frontsite/PatronProfile/PatronPastLoans/PatronPastLoans.js]:
```suggestion
        messageContent={"You do not have any past loan."}
```

File: ui/src/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js [hunk 6]
```diff
@@ -68,7 +68,7 @@ export default class PatronPendingLoans extends Component {
     return (
       <NoResultsMessage
         messageHeader={'No loan requests'}
-        messageContent={'Currently you do not have any loan requests'}
+        messageContent={"You don't currently have any loan request."}
```
Comment (topless) [inline ui/src/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js]:
```suggestion
        messageContent={"You do not have any loan request."}
```

**Review by topless — APPROVED** [review]

File: ui/src/pages/frontsite/PatronProfile/PatronMetadata/PatronMetadata.js (hunk 4 above)
Comment (ntarocco) [reply ui/src/pages/frontsite/PatronProfile/PatronMetadata/PatronMetadata.js]: Not sure about this, any other opinion from any one else? I will actually remove the sentence.

File: ui/src/pages/frontsite/PatronProfile/PatronPastDocumentRequests/PatronPastDocumentRequests.js (hunk 1 above)
Comment (ntarocco) [reply ui/src/pages/frontsite/PatronProfile/PatronPastDocumentRequests/PatronPastDocumentRequests.js]: Fixed! 👍

File: ui/src/pages/frontsite/PatronProfile/PatronMetadata/PatronMetadata.js (hunk 4 above)
Comment (ntarocco) [reply ui/src/pages/frontsite/PatronProfile/PatronMetadata/PatronMetadata.js]: Removed

**Pull request merged**